Pages/Shared/layout.html

```diff
@@ -1,22 +1,38 @@
+<!doctype html>
 <html>
 <head>
     <title>Template</title>
-    <script src="../../Libraries/jquery-3.5.1.min.js"></script>
-    <script>
-        function check() {
-            $('#tar').load('error.html');  // here 'test.html' is a page and 'target' id a id of 'test.html'
-        }
-    </script>
+    <script src="../../Scripts/jquery-3.5.1.min.js"></script>
+    <script src="../../Scripts/bootstrap.min.js"></script>
+    <link href="../../Content/bootstrap.min.css" rel="stylesheet" />
+
+    <script src="../../Scripts/popper.js"></script>
+    <link href="../../Styles/main.css" rel="stylesheet" />
+
 </head>
-  <body>
-<input type="button" id="btn" value="Go"  onclick="check();"/>
-<div>
-    <h1>Header</h1>
-</div>
-<div id="tar">
-</div>
-<div>
-   Footer
-</div>
+<body>
+
```

Comment:
No need of space. Adding spaces will cause the page to take time to load which affects the performance.